galley/pkg/config/processing/snapshotter/snapshotter.go

```diff
@@ -85,8 +85,12 @@ func (a *accumulator) Handle(e event.Event) {
 		monitoring.RecordStateTypeCount(e.Source.String(), a.collection.Size())
 		monitorEntry(e.Source, e.Entry.Metadata.Name, false)
 
+	case event.Disabled:
+		a.collection.Disable()
```

Comment:
Does this need to count against syncCount?

Reply:
In the current Disabled semantics, no, since in this version Disabled does not replace FullSync. 

If we change the semantics so that sending Disabled means you don't send FullSync, then yes this should be updated.

Reply:
Got rid of this, obsolete.